unlock-app/src/services/storageService.js

```diff
@@ -29,6 +30,7 @@ export const failure = {
   getUserPrivateKey: 'getUserPrivateKey.failure',
   getUserRecoveryPhrase: 'getUserRecoveryPhrase.failure',
   getCards: 'getCards.failure',
+  keyPurchase: 'keyPurchase.success',
```

Comment:
```suggestion
  keyPurchase: 'keyPurchase.failure',
```

Reply:
That's what I get for lazy copy-pasting! Good catch